libsolidity/formal/SMTChecker.cpp

```diff
@@ -889,7 +906,8 @@ SMTChecker::checkSatisfiableAndGenerateModel(vector<smt::Expression> const& _exp
 		try
 		{
 			// Parse and re-format nicely
-			value = formatNumber(bigint(value));
+			if (isValidDecimal(value) || isValidHex(value))
```

Comment:
This seems to be an unrelated change?

If so would split it into a separate PR or at least separate commit.

Reply:
Removed